### PR TITLE
SE Metrics Guide 

### PR DESCRIPTION
**Pull request by pfmackin**
This is the SE Metrics Guide. It has several examples that demonstrate how to use Helidon metrics for a Helidon SE based application.

### CODE REVIEW

File: docs/src/main/docs/guides/09_metrics_se_guide.adoc [hunk 1]
```diff
@@ -0,0 +1,858 @@
...
+
+[source,bash]
+----
+NAME             TYPE       CLUSTER-IP      EXTERNAL-IP   PORT(S)          AGE
+helidon-metrics   NodePort   10.99.159.2   <none>        8080:31143/TCP   8s // <1>
+----
+<1> A service of type `NodePort` that serves the default routes on port `31143`
+
+[source,bash]
+.Verify the metrics endpoint using port `30116`, your port will likely be different:
```
Comment (barchetta) [inline docs/src/main/docs/guides/09_metrics_se_guide.adoc]: 31143 ?

File: docs/src/main/docs/guides/09_metrics_se_guide.adoc [hunk 2]
```diff
@@ -0,0 +1,858 @@
...
+This guide describes how to create a sample Helidon SE project
+that can be used to run some basic examples using both built-in and custom metrics.
+
+== What you need
+
+[width=50%,role="flex, sm7"]
+|===
+|About 30 minutes
+|<<about/03_prerequisites.adoc,Helidon Prerequisites>>
+|===
```
Comment (barchetta) [inline docs/src/main/docs/guides/09_metrics_se_guide.adoc]: Add helm

Comment (pfmackin) [reply docs/src/main/docs/guides/09_metrics_se_guide.adoc]: done

File: docs/src/main/docs/guides/09_metrics_se_guide.adoc (hunk 1 above)
Comment (pfmackin) [reply docs/src/main/docs/guides/09_metrics_se_guide.adoc]: fixed

**Pull request merged**